aws_cloudwatch.go

```diff
@@ -26,17 +26,21 @@ type cloudwatchInterface struct {
 }
 
 type cloudwatchData struct {
-	ID                     *string
-	Metric                 *string
-	Service                *string
-	Statistics             []string
-	Points                 []*cloudwatch.Datapoint
-	NilToZero              *bool
-	AddCloudwatchTimestamp *bool
-	CustomTags             []tag
-	Tags                   []tag
-	Dimensions             []*cloudwatch.Dimension
-	Region                 *string
+	ID                      *string
+	MetricId                *string
```

Comment:
In keeping with struct field naming practices I'd recommend we rename this to `MetricID`

Reply:
fixed